web_modules/Wrapper/index.css

```diff
@@ -16,18 +17,18 @@
 :global .markdownIt-Anchor {
   position: absolute;
   left: -1.5rem;
-  opacity: .1;
-  transition: opacity .2s;
+  opacity: 0.1;
+  transition: opacity 0.2s;
 }
 
-  :global h1:hover .markdownIt-Anchor,
-  :global h2:hover .markdownIt-Anchor,
-  :global h3:hover .markdownIt-Anchor,
-  :global h4:hover .markdownIt-Anchor,
-  :global h5:hover .markdownIt-Anchor,
-  :global h6:hover .markdownIt-Anchor {
-    opacity: 1;
-  }
+:global h1:hover .markdownIt-Anchor,
+:global h2:hover .markdownIt-Anchor,
+:global h3:hover .markdownIt-Anchor,
+:global h4:hover .markdownIt-Anchor,
+:global h5:hover .markdownIt-Anchor,
+:global h6:hover .markdownIt-Anchor {
+  opacity: 1;
+}
```

Comment:
Yep, I believe the ['hierarchicalSelectors option'](https://github.com/stylelint/stylelint/blob/master/src/rules/indentation/README.md#hierarchicalselectors-truefalse) within the `indentation` rule would work here. But I figured it'd be better to use the actual nesting feature of `postcss-cssnext` and not have to extend and override the `indentation` rule within `stylelint-config-standard`. The CSS [now looks like](https://github.com/postcss/postcss.org/blob/master/web_modules/Wrapper/index.css#L25):

```css
:global .markdownIt-Anchor {
  position: absolute;
  left: -1.5rem;
  opacity: 0.1;
  transition: opacity 0.2s;

  @nest :--headings:hover & {
    opacity: 1;
  }
}
```

It was changed in the [follow-up PR](https://github.com/postcss/postcss.org/pull/55) that added `postcss-cssnext` to the boilerplate.

Sound good to you, or shall we adapt this bit of code?